coalib/results/Result.py

```diff
@@ -264,6 +264,55 @@ def location_repr(self):
         return ', '.join(repr(relpath(range_path))
                          for range_path in sorted(range_paths))
 
+    @enforce_signature
+    def get_affected_source_lines(self,
```

Comment:
`get_affected_source_lines` uses the contents of a file. `SourceRange` class only has the information about line numbers and file name and not about contents of a file, so I can not make that as a property.

Probably I will move that function to `SourceRange` class and pass file content information to the file instead of start, end and filename.